### PR TITLE
UNIFY-653-rename-integration-folder

### PR DESCRIPTION
**Pull request by jaffrepaul**
https://cypress-io.atlassian.net/browse/UNIFY-653

### CODE REVIEW

File: content/guides/references/legacy-configuration.md [hunk 1]
```diff
@@ -73,7 +73,6 @@ you should understand well. The default values listed here are meaningful.
...
 | `fixturesFolder`    | `cypress/fixtures`         | Path to folder containing fixture files (Pass `false` to disable)                                                                                                                                                                                                                          |
 | `ignoreTestFiles`   | `*.hot-update.js`          | A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. Cypress uses `minimatch` with the options: `{dot: true, matchBase: true}`. We suggest using [https://globster.xyz](https://globster.xyz) to test what files would match. |
-| `integrationFolder` | `cypress/integration`      | Path to folder containing integration test files                                                                                                                                                                                                                                           |
```
Comment (cowboy) [inline content/guides/references/legacy-configuration.md:76]: Please revert this change

File: .circleci/config.yml [hunk 2]
```diff
@@ -131,7 +131,7 @@ workflows:
       - cypress/run:
           name: 'UI Tests - Chrome'
           browser: chrome
-          spec: cypress/integration/*
+          spec: cypress/e2e/*
```
Comment (cowboy) [inline .circleci/config.yml:134]: I expected changing `spec` without also changing the location of the E2E tests would break the build, but it didn't, probably because we have a custom `command` defined and the project [cypress.json](https://github.com/cypress-io/cypress-documentation/blob/master/cypress.json) `testFiles` is set to `**/*.spec.js`.

Can you try removing this `spec` line and see if it still builds? If this config isn't used, we should remove it.

File: content/guides/references/migration-guide.md [hunk 3]
```diff
@@ -1294,7 +1294,7 @@ result of each test retry.
       "state": "failed",
       "body": "function () {\n  expect(true).to.be["false"];\n}",
       "stack": "AssertionError: expected true to be false\n' +
-        '    at Context.eval (...cypress/integration/spec.js:5:21",
+        '    at Context.eval (...cypress/e2e/spec.js:5:21",
```
Comment (cowboy) [inline content/guides/references/migration-guide.md:1297]: Please revert all changes in the migration guide

**Review by cowboy — CHANGES_REQUESTED** [review]

File: content/guides/references/legacy-configuration.md (hunk 1 above)
Comment (jaffrepaul) [reply content/guides/references/legacy-configuration.md:76]: Easy to get overzealous with these changes 😄

**Review by cowboy — APPROVED** [review]
